lib/MusicBrainz/Server/Data/WebService.pm

```diff
@@ -201,11 +201,11 @@ sub xml_search
     }
 
     $query =~ s/^ AND //;
-    # In case we have a blank query
-    if ($query =~ /^\s*$/)
+    # In case we have a blank query (only whitespace, or whitespace surrounded by unescaped quotes)
+    if ($query =~ /^['"]?\s*['"]?$/)
```

Comment:
Lucene doesn't support single quotes for quoting things, so I'd remove those. The double quotes also have to be balanced, so I'd do something like this instead (untested):
```suggestion
    if ($query =~ /^(?:\s*|"\s*")$/)
```